docs/experience/user-identification.md

```diff
@@ -0,0 +1,21 @@
+title: Identifying Users
+description: Overview on how to identify users
+
+By default, all users that interact with your application monitored by Sematext Experience will be anonymous. You will get an overview of the number of sessions for the given user, page load and AJAX events, User Satisfaction for those events, used browser and the country from which the user is interacting with your site. 
```

Comment:
`AJAX`, was just talking about this with @adnanrahic, we should use "network requests" instead. He complained it's too 2012 :)

Reply:
Done :)